src/main/scala/firrtl/ExecutionOptionsManager.scala

```diff
@@ -215,8 +225,17 @@ trait HasFirrtlOptions {
     foreach { x =>
       firrtlOptions = firrtlOptions.copy(outputFileNameOverride = x)
     }.text {
-      "use this to override the default name, default is empty"
-    }
+    "use this to override the default output file name, default is empty"
+  }
+
+  parser.opt[String]("annotation-file")
+    .abbr("a")
+    .valueName ("<output>").
+    foreach { x =>
+      firrtlOptions = firrtlOptions.copy(outputFileNameOverride = x)
+    }.text {
+    "use this to override the default annotation file name, default is empty"
+  }
```

Comment:
Its something to think about, but I think this should be in FirrtlOptions, at least until we think chisel should accept an annotation file as a argument (which we haven't planned on yet).